framework/test/unit/modules/pos/method.spec.ts

```diff
@@ -41,6 +41,7 @@ describe('PoSMethod', () => {
 	let validatorSubStore: ValidatorStore;
 	let nameSubStore: NameStore;
 	const address = utils.getRandomBytes(20);
+	const invalidAddress = utils.getRandomBytes(0);
```

Comment:
If you wanted to generate an empty Buffer to use as an invalid address, there is a faster method that we've been using throughout the codebase:

```suggestion
	const invalidAddress = Buffer.alloc(0);
```

Reply:
Actually this could be any invalid adress `const invalidAddress = utils.getRandomBytes(1000);` but yes, for invalid empty buffers, am all for the fastest way 😎.